loop_lab.js

```diff
@@ -0,0 +1,173 @@
+// // 1. Write a `while` loop and a `for` loop that takes the variable
+// // "num" and logs all the numbers, descending, between "num" and 1.
+//
+let num = 10
+while (num >= 0){
+  console.log(num)
+  num --
+};
+
+for(let num1 = 20; num1 >= 10; num1 -- ) {
+  console.log(num1)
+};
+//
+// //------------------------------
+// //2. Write a `while` loop and a `for` loop that takes the variable "max",
+// //and iterates over all numbers from 0 to "max".
+// //For each iteration, it will check if the current number is even or odd,
+// //and log that to the screen (e.g. "2 is even")
+//
+let max = 35
```

Comment:
This code wouldn't work if max was changed to 21 or any value less than 30. How can we fix this?